src/Parser/RestoDruid/Modules/Features/StatWeights.js

```diff
@@ -0,0 +1,316 @@
+      console.log(`Crit - ${formatNumber(this.totalOneCrit)}`);
+      console.log(`Haste HPM - ${formatNumber(this.totalOneHasteHpm)}`);
+      console.log(`Mastery - ${formatNumber(this.totalOneMastery)}`);
+      console.log(`Vers - ${formatNumber(this.totalOneVers)}`);
+      console.log(`Leech - ${formatNumber(this.totalOneLeech)}`);
+    }
+  }
+
+  _ratingPerOnePercent(oneRatingHealing) {
+    const onePercentHealing = this.totalAdjustedHealing / 100;
+    return onePercentHealing / oneRatingHealing;
+  }
+
+  _prepareResults() {
+    const hasLeech = this.combatants.selected.leechRating > 0;
+
+    const intWeight = this.totalOneInt / this.totalOneInt;
+    const critWeight = this.totalOneCrit / this.totalOneInt;
+    const hasteHpmWeight = this.totalOneHasteHpm / this.totalOneInt;
+    const masteryWeight = this.totalOneMastery / this.totalOneInt;
+    const versWeight = this.totalOneVers / this.totalOneInt;
+    let leechWeight = this.totalOneLeech / this.totalOneInt;
+    if(!hasLeech) {
+      leechWeight = "NYI";
+    }
+
+    const intForOnePercent = this._ratingPerOnePercent(this.totalOneInt);
+    const critForOnePercent = this._ratingPerOnePercent(this.totalOneCrit);
+    const hasteHpmForOnePercent = this._ratingPerOnePercent(this.totalOneHasteHpm);
+    const masteryForOnePercent = this._ratingPerOnePercent(this.totalOneMastery);
+    const versForOnePercent = this._ratingPerOnePercent(this.totalOneVers);
+    let leechForOnePercent = this._ratingPerOnePercent(this.totalOneLeech);
+    if(!hasLeech) {
+      leechForOnePercent = "NYI";
+    }
+
+    const hasteHpmTooltip = "HPM stands for 'Healing per Mana'. In valuing Haste, it considers only the faster HoT ticking and not the reduced cast times. Effectively it models haste's bonus to mana efficiency. This is typically the better calculation to use for raid encounters where mana is an issue.";
+    const leechTooltip = "Leech weight can currently only be calculated when player already has some Leech rating";
+
+    return [
+      { stat:'Intellect', weight:intWeight, ratingForOne:intForOnePercent },
+      { stat:'Crit', weight:critWeight, ratingForOne:critForOnePercent },
+      { stat:'Haste (HPM)', weight:hasteHpmWeight, ratingForOne:hasteHpmForOnePercent, tooltip:hasteHpmTooltip },
+      { stat:'Mastery', weight:masteryWeight, ratingForOne:masteryForOnePercent },
+      { stat:'Versatility', weight:versWeight, ratingForOne:versForOnePercent },
+      { stat: 'Leech', weight:leechWeight, ratingForOne:leechForOnePercent, tooltip:leechTooltip },
+    ];
+  }
+
+  extraPanel() {
+    const results = this._prepareResults();
+    return (
+      <div className="panel items">
+      <div className="panel-heading">
+        <h2><dfn data-tip="Weights are calculated using the actual circumstances of this encounter. Weights are likely to differ based on fight, raid size, items used, talents chosen, etc.">STAT WEIGHTS</dfn>
```

Comment:
Better not to capitalize; it's currently a design choice that's taken care of by CSS. It might change in the future.

Reply:
Fixing

Reply:
Jk it looks dumb as hell and is out of style with other resto tooltips, not fixing